apps/src/StudioApp.js

```diff
@@ -730,6 +730,13 @@ export function makeFooterMenuItems() {
       newWindow: true
     }
   ];
+
+  //Removes 'Try-HOC' from only gamelab footer menu
+  if (project.getProjectUrl().includes('gamelab')) {
+    //Look for: More elegant way to do this. -EP
+    footerMenuItems.shift();
```

Comment:
I can think of a few ways we can make this more elegant.

* One is eliminating the gamelab special case here altogether, by giving gamelab its own version of this function.  That makes some sense, since applab already has one - but then, it'd be nice to deduplicate that code somewhat too.    Maybe we can brainstorm on this together.
* Two is a nicer way to tell that we're in gamelab in the first place.  Instead of checking the project URL, [the `appOptions` global has an `app` property](https://github.com/code-dot-org/code-dot-org/blob/9d553e125a45dfa0524bbaa0182901ae964f9cc1/apps/src/util/unsupportedBrowserWarning.js#L16-L16) that is probably a better way to check this.  This global `appOptions` object also gets passed around the initialization process as `config` and along the way the app type gets [added to the redux `pageConstants` module](https://github.com/code-dot-org/code-dot-org/blob/2bff45b95137a1c85fa85cc6b3c08d927d652a11/apps/src/StudioApp.js#L2762-L2762) - I think that's probably the best place to get it at this point, unless you're very early in the initialization process.